src/mudlet-lua/lua/Other.lua

```diff
@@ -3,7 +3,7 @@
 ----------------------------------------------------------------------------------
 
 mudlet = mudlet or {}
-mudlet.supportscoroutines
+mudlet.supportscoroutines = true
```

Comment:
And get the C++ core to wipe/regenerate the `mudlet` table as part of the `luaGlobal.lua` code? Is it as simple as that (apart from warning the user that that table will NEVER be preserved even if they check it to be saved in future)?

It now will store the `keycode` and `keymodifier` tables going forward from this point but what other purpose did/does this particular table serve?